common/pom.xml

```diff
@@ -14,6 +14,12 @@
             <groupId>org.opennms.integration.api</groupId>
             <artifactId>api</artifactId>
         </dependency>
+        <!-- TODO MVR can we just add this without issues? -->
```

Comment:
I wouldn't bring in Guava. We already set a precedent for how to write the immutable implementations without any thirdparty libraries. We just make a copy of the collection and wrap it as unmodifiable.

Reply:
Yeah, good point. Originally I added it to get the original code working. But guava isn't really needed anymore.